packages/standard-components/src/forms/InnerForm.svelte

```diff
@@ -108,8 +116,28 @@
 
       return !newError
     }
+
+    const clearValue = () => {
+      const { fieldState } = fieldMap[field]
+      fieldState.update(state => {
+        state.value = defaultValue
+        state.error = null
+        return state
+      })
+
+      formState.update(state => {
+        state.values = { ...state.values, [field]: defaultValue }
```

Comment:
Using the new variable declared above, we can update this to:
`state.values = { ...state.values, [field]: newValue }`
to accound for initial values and default values.